app/controllers/issues_controller.rb

```diff
@@ -22,7 +23,7 @@ def new
   end
 
   def show
-    @issue = Issue.find(params[:id])
+    @voted = @issue.votes.where(user_id: current_user.id)
     load_relevant_issues
     redirect_back(fallback_location: root_path) unless \
                                                 @issue.can_read?(current_user)
```

Comment:
@Kmalynss I moved `@issue = Issue.find(params[:id])` to private method init_issue, because there was too much logic inside one of methods.
I use `@voted` only to check, if the user with `current_user.id` has already voted for the @issue.